arkouda-docker/arkouda-udp-server

```diff
@@ -0,0 +1,69 @@
+FROM hokiegeek2/chapel-gasnet-udp:1.27.0
```

Comment:
This won't exist for other users, we might want to specify the need for updating this in the `README.md` or maybe better to update it to use an environment variable so it doesn't need to be updated by anyone trying to use it

Reply:
@joshmarshall1 Gotcha, yeah, as I noted above, I am hoping to establish a Bears-R-Us repo and move all these images in there.

Reply:
I don't know too awful much about docker, but I believe this point to @hokiegeek2's image on docker hub https://hub.docker.com/r/hokiegeek2/chapel-gasnet-udp